### PR TITLE
NAS-126774 / 24.10 / Add aria-label for ix-forms module inputs

### PR DESCRIPTION
**Pull request by RehanY147**

### CODE REVIEW

**Review by undsoft — CHANGES_REQUESTED** [review]
Looks good, but let's do this for other ix-controls.

File: src/app/modules/ix-forms/components/ix-explorer/ix-explorer.component.html [hunk 1]
```diff
@@ -33,7 +33,7 @@
     {{ 'Create Dataset' | translate }}
   </button>
```
Comment (undsoft) [inline src/app/modules/ix-forms/components/ix-explorer/ix-explorer.component.html:34]: Also needs label on the input above.

**Review by undsoft — APPROVED** [review]
Looks pretty good. Please also check why Safari thinks that normal inputs (i.e. is SMB for are treated as autocompletes). It also reads some extra about stale data or something similar.

File: scripts/ui/ui.ts [hunk 2]
```diff
@@ -711,6 +711,7 @@ function mockPool(command: Command, options: CommandOptions): void {
 * Remote Command
 * */
 function normalizeUrl(url = ''): string {
+  return url;
```
[the remark this answers is not in the document]
Comment (RehanY147) [reply scripts/ui/ui.ts:714]: Whoops

**Comment by RehanY147** [comment]
backport

**Pull request merged**